### ISSUE TITLE
Various Upgrading to 0.3.1 Admin UI issues

### ISSUE THREAD

**Issue by AndersonTaylor** [keystonejs/keystone#1098]
Just upgraded to version 3.1 from 2.36 and running into a few issues with the admin ui.

ONE:    Now whenever I try to pick a date (either manually or using the Today button) 
    I get an unresponsive script and everything freezes.

```
Script: http://localhost:3000/keystone/build/js/app.js:8
```

TWO:     Empty Numerical  (not required)  fields are not validating :

```
There was a problem saving your changes:
    Please enter a valid number in the Quantity field
```

**Comment by JohnnyEstilles** [comment]
@AndersonTaylor, thanks for reporting these issues.

Re: ONE - See issues #1038, #1061 and #1065   - a discussion on the subject is already in progress.

Re: TWO - Can you provide some code, maybe a snippet of the model in which you're having the issue?

**Comment by AndersonTaylor** [comment]
Thanks for the info on ONE.

For TWO I can reproduce it on a fresh keystone project
(using yo keystone) just by adding this to the User model:
```
quantity: { type: Types.Number, label: 'Quantity' }
```
and then saving the user with the field blank.


On OSX       node version 0.12.0

**Comment by JedWatson** [comment]
These have both been addressed, thanks of the report @AndersonTaylor.

The date field picker was addressed in 0.3.2, and I've just fixed the issue with Number fields, will be released with 0.3.3 (both fixed in `master`).
